ci/install.sh

```diff
@@ -16,7 +16,7 @@ Options:
     --crate NAME    Name of the crate to install (default <repository name>)
     --tag TAG       Tag (version) of the crate to install (default <latest release>)
     --target TARGET Install the release compiled for $TARGET (default <`rustc` host>)
-    --to LOCATION   Where to install the binary (default ~/.cargo/bin)
+    --to LOCATION   Where to install the binary (default '$CARGO_HOME' if defined, otherwise '~/.cargo/bin')
```

Comment:
```suggestion
    --to LOCATION   Where to install the binary (default '$CARGO_HOME/bin' if defined, otherwise '~/.cargo/bin')
```

More context is in the comment below.